resources/js/Pages/Vault/Contact/Move.vue

```diff
@@ -0,0 +1,119 @@
+<script setup>
+import Layout from '@/Shared/Layout.vue';
+import { useForm } from '@inertiajs/inertia-vue3';
+
+const props = defineProps({
+  layoutData: Object,
+  data: Object,
+});
+
+const form = useForm({
+  other_vault_id: 0,
+});
+
+const move = (vault) => {
+  form.other_vault_id = vault.id;
+
+  form.post(props.data.url.move, {
+    onFinish: () => {},
+  });
```

Comment:
It doesn't worth adding this empty function
```suggestion
  form.post(props.data.url.move);
```